docs/design/proposals/rbd-volume-healer.md

```diff
@@ -0,0 +1,94 @@
+# RBD NBD VOLUME HEALER
+
+- [RBD NBD VOLUME HEALER](#rbd-nbd-volume-healer)
+  - [Rbd Nbd](#rbd-nbd)
+    - [Advantages of userspace mounters](#advantages-of-userspace-mounters)
+    - [Side effects of userspace mounters](#side-effects-of-userspace-mounters)
+  - [Volume Healer](#volume-healer)
+    - [More thoughts](#more-thoughts)
+
+## Rbd Nbd
```

Comment:
You have a mix of Rbd, RBD and rbd. Maybe settle on one form of capitalization? Same for NBD.

Reply:
OK